modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/resources/META-INF/resources/admin/export_record_set.jspf

```diff
@@ -13,16 +13,31 @@
  * details.
  */
 --%>
+<%
+String warning = "show-warning";
+
+if(Validator.isNotNull(ddlFormAdminDisplayContext.getShowCSVWarning())) {
+	warning = ddlFormAdminDisplayContext.getShowCSVWarning();
+}
+
+boolean showWarning = StringUtil.equals("show-warning", warning);
```

Comment:
Rename variable to `showCSVWarning`.
Basic idea: Add "csv" to most of the variable names.